src/it/junit/issue_50-empty-tags/pom.xml

```diff
@@ -42,9 +42,14 @@
                             <goal>generateRunners</goal>
                         </goals>
                         <configuration>
-                            <glue>foo, bar</glue>
-                            <!-- No tags -->
-                            <tags></tags>
+                            <glue>
+                                <package>foo</package>
+                                <package>bar</package>
+                            </glue>
+                            <!-- Empty tags -->
+                            <tags>
+                                <tag></tag>
+                            </tags>
```

Comment:
No tags was no longer a problem. Renamed the issue to empty tags which is the list based equivalent.